<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutWindow</class>
 <widget class="QDialog" name="AboutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="aboutWinClose">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="aboutWinTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="recordsTab">
    <attribute name="title">
     <string>Records</string>
    </attribute>
    <widget class="QLabel" name="aboutWinRecordsTitle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Records</string>
     </property>
    </widget>
    <widget class="QLabel" name="aboutWinRecordsText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>540</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tagsTab">
    <attribute name="title">
     <string>Tags</string>
    </attribute>
    <widget class="QLabel" name="aboutWinTagsTitle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
    <widget class="QLabel" name="aboutWinTagsText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>540</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="discogsTab">
    <attribute name="title">
     <string>Discogs</string>
    </attribute>
    <widget class="QLabel" name="aboutWinDiscogsTitle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Discogs</string>
     </property>
    </widget>
    <widget class="QLabel" name="aboutWinDiscogsText">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>540</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>5</number>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
